Example/URBNDataSource/ViewControllers/Code Accordion CollectionView/URBNAccordionCollectionVC.m

```diff
@@ -0,0 +1,139 @@
+@interface URBNAccordionCVHeader : UICollectionReusableView
+
+@property (nonatomic, strong) UILabel *catLabel;
+@property (nonatomic, assign) BOOL expanded;
+@property (nonatomic, copy) void (^tappedBlock)();
+@property (nonatomic, strong) UIView *line;
+@property (nonatomic, strong) UIImageView *expandedImgV;
+- (void)setExpanded:(BOOL)expanded;
+
+@end
+
+@implementation URBNAccordionCVHeader
+
+- (instancetype)initWithFrame:(CGRect)frame {
+    self = [super initWithFrame:frame];
+    if (self) {
+        self.backgroundColor = [UIColor whiteColor];
+        
+        self.catLabel = [UILabel new];
+        self.catLabel.font = [UIFont systemFontOfSize:14.0];
+        self.catLabel.textColor = [UIColor blueColor];
+        self.catLabel.translatesAutoresizingMaskIntoConstraints = NO;
+        [self.catLabel setContentHuggingPriority:UILayoutPriorityDefaultLow forAxis:UILayoutConstraintAxisVertical];
+        [self addSubview:self.catLabel];
+        [self addConstraints:[NSLayoutConstraint constraintsWithVisualFormat:@"H:|-15-[_catLabel]" options:0 metrics:nil views:NSDictionaryOfVariableBindings(_catLabel)]];
+        [self addConstraints:[NSLayoutConstraint constraintsWithVisualFormat:@"V:|[_catLabel]|" options:0 metrics:nil views:NSDictionaryOfVariableBindings(_catLabel)]];
+        
+        self.expandedImgV = [[UIImageView alloc] initWithImage:[UIImage imageNamed:@"shop-category-plus"]];
+        self.expandedImgV.highlightedImage = [UIImage imageNamed:@"shop-category-minus"];
+        self.expandedImgV.translatesAutoresizingMaskIntoConstraints = NO;
+        self.expandedImgV.contentMode = UIViewContentModeScaleAspectFit;
+        [self addSubview:self.expandedImgV];
+        [self addConstraints:[NSLayoutConstraint constraintsWithVisualFormat:@"H:[_expandedImgV]-15-|" options:0 metrics:nil views:NSDictionaryOfVariableBindings(_expandedImgV)]];
+        [self addConstraints:[NSLayoutConstraint constraintsWithVisualFormat:@"V:|[_expandedImgV]|" options:0 metrics:nil views:NSDictionaryOfVariableBindings(_expandedImgV)]];
+        
+        UITapGestureRecognizer *tap = [[UITapGestureRecognizer alloc] initWithTarget:self action:@selector(tapped:)];
+        tap.numberOfTapsRequired = 1;
+        tap.numberOfTouchesRequired = 1;
+        [self addGestureRecognizer:tap];
+        
+        self.line = [UIView new];
+        self.line.backgroundColor = [UIColor lightGrayColor];
+        self.line.translatesAutoresizingMaskIntoConstraints = NO;
+        [self addSubview:self.line];
+        [self addConstraints:[NSLayoutConstraint constraintsWithVisualFormat:@"H:|[_line]|" options:0 metrics:nil views:NSDictionaryOfVariableBindings(_line)]];
+        [self addConstraints:[NSLayoutConstraint constraintsWithVisualFormat:@"V:[_line(==0.5)]|" options:0 metrics:nil views:NSDictionaryOfVariableBindings(_line)]];
+    }
+    
+    return self;
+}
+
+- (void)tapped:(UITapGestureRecognizer *)tap {
+    if (UIGestureRecognizerStateEnded == tap.state) {
```

Comment:
You should check *if* tappedBlock exists.   Since this is a public pod you want to show best practices